pkg/u8proto/u8proto.go

```diff
@@ -62,5 +63,13 @@ func ParseProtocol(proto string) (U8proto, error) {
 	if u, ok := ProtoIDs[strings.ToLower(proto)]; ok {
 		return u, nil
 	}
-	return 0, fmt.Errorf("unknown protocol '%s'", proto)
+	return 0, fmt.Errorf("unknown protocol %q", proto)
```

Comment:
The change from `"%s"` to `%q` is causing the unit tests to fail.

Reply:
Fixed the unit tests. This should be a quote.